tests/test-recipes/metadata/_overlinking_detection_ignore_patterns/build_scripts/with_bzip2.bat

```
@@ -0,0 +1,5 @@
+@echo on
+echo #include ^<bzlib.h^>>main.c
```

Ugh, windows cmd quoting 🤦🏼‍♂️

---

yeah. To be clear, that is just a straight copy from the existing _overlinking test. It's just adapted here to ignore the error/warning that normally shows up.

---

gotcha, good to know